build.gradle.kts

```diff
@@ -157,24 +153,24 @@ subprojects {
     JavadocConfig.applyTo(project)
     CheckStyleConfig.applyTo(project)
 
+    val javaVersion = 8
```

Comment:
Please use the constant instead of a literal. Detekt is not happy.

Reply:
As discussed, there is no good solution on converting between `JavaVersion.VERSION_1_8` and `int 8`, therefore, this warning will be suppressed.